modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/utils.es.js

```diff
@@ -97,6 +78,31 @@ export function hasConfigBreakChanges({draftConfig, ...config}) {
 		.some((isDifferent) => isDifferent);
 }
 
+export function populateFormViewFields(formView) {
+	return {
+		...formView,
+		fields: formView.dataLayoutPages.reduce(
```

Comment:
@jotascript we need to use optional chaining in this case `formView.dataLayoutPages?.reduce`. This wasn't necessary before because the dataLayoutPages was receiving an initial value (dataLayoutPages = [])